bats/tests/helpers/utils.bash

```diff
@@ -360,6 +360,12 @@ capture_logs() {
         local logdir
         logdir=$(unique_filename "${PATH_BATS_LOGS}/${RD_TEST_FILENAME}")
         mkdir -p "$logdir"
+        # On Linux/macOS, the symlinks to the lima logs might be dangling.
+        # Remove any dangling ones before doing the copy.
+        find -L "${PATH_LOGS}/" -type l \
+            -exec rm -f '{}' ';' \
+            -exec touch '{}' ';' \
+            -exec echo 'Removed dangling symlink' '{}' ';'
```

Comment:
Not sure why you need the `touch`? Just so you don't wonder why the file is missing?

```suggestion
            -exec touch '{}' ';' \
            -exec echo 'Replaced dangling symlink with empty file' '{}' ';'
```

Reply:
Yeah, this way we get a zero-byte file in the archive, so we won't be confused that it's missing.